pkg/services/ldap/ldap.go

```diff
@@ -397,7 +412,7 @@ func (server *Server) Auth(username, password string) error {
 
 // AuthAdmin authentificates LDAP admin user
 func (server *Server) AuthAdmin() error {
```

Comment:
Makes sense to remove this method as well?

Reply:
it does, but I don't want to do that refactoring this late in the game. We'll have to clean that up later